src/screens/TopicPageScreen/index.js

```diff
@@ -407,30 +419,35 @@ const TopicPageScreen = (props) => {
         isFollow={isFollow}
         getTopicDetail={getTopicDetail}
       />
-      <TiktokScroll
-        ref={listRef}
-        contentHeight={dimen.size.TOPIC_CURRENT_ITEM_HEIGHT + normalizeFontSizeByWidth(4)}
-        data={feeds}
-        onEndReach={onEndReach}
-        onRefresh={onRefresh}
-        refreshing={loading}
-        renderItem={renderItem}
-        onScroll={handleScroll}
-        scrollEventThrottle={16}
-        showSearchBar={true}
-        searchHeight={isHeaderHide ? headerHideHeight : headerShowHeight}
-        snap
-        contentOffset={{x: 0, y: topPosition}}
-        contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
-      />
+      <View
+        style={{
+          marginTop: isHeaderHide ? -dimen.size.TOPIC_FEED_HEADER_HEIGHT : 0,
+          minHeight: 548,
+          height: 700
+        }}>
+        <TiktokScroll
+          ref={listRef}
+          contentHeight={dimen.size.TOPIC_CURRENT_ITEM_HEIGHT + normalizeFontSizeByWidth(4)}
+          data={feeds}
+          onEndReach={onEndReach}
+          onRefresh={onRefresh}
+          refreshing={loading}
+          renderItem={renderItem}
+          onScroll={handleScroll}
+          scrollEventThrottle={16}
+          showSearchBar={true}
+          snap
+          contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
+        />
+      </View>
       <ButtonAddPostTopic topicName={topicName} onRefresh={onRefresh} />
       <BlockComponent
         ref={refBlockComponent}
         refresh={onBlockCompleted}
         refreshAnonymous={onDeleteBlockedPostCompleted}
         screen="topic_screen"
       />
-    </SafeAreaProvider>
+    </>
   );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `SafeAreaProvider` has been removed and replaced with a `View` component. This could potentially lead to layout issues on devices with notches or safe area insets. Also, the `withInteractionsManaged` HOC has been removed from the export statement. Ensure that this does not affect the interaction handling of the component.
```diff
- <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
+ <View
+   style={{
+     marginTop: isHeaderHide ? -dimen.size.TOPIC_FEED_HEADER_HEIGHT : 0,
+     minHeight: 548,
+     height: 700
+   }}>
- export default withInteractionsManaged(TopicPageScreen);
+ export default TopicPageScreen;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->